msg/vehicle_constraints.msg

```diff
@@ -0,0 +1,13 @@
+# Local setpoint constraints in NED frame
+# setting something to NaN means that no limit is provided
+
+float32 yawspeed	# in radians/sec
+float32 speed_xy	# in meters/sec
+float32 speed_up	# in meters/sec
```

Comment:
the setpoint which gets constrained is called `vz` so why not call the constraint `constraint.vz_up`?